Lil-Peep/src/styles/modules/description-banner.scss

```diff
@@ -0,0 +1,29 @@
+.banner_section {
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.banner_content {
+  background: linear-gradient(90deg,
+    rgba(200, 190, 183, 1 ),
+    rgba(200, 190, 183, 1) 30%,
+    rgba(200, 190, 183, 1) 30%,
+    rgba(200, 190, 183, 0));
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.